### DISCUSSION TITLE
DecimalEditor and Culture ru-Ru

### DISCUSSION THREAD

**Discussion by Zahar661** [serenity-is/Serenity#6571] [Help]
Good afternoon! When entering an amount greater than 999, the validator generates an error due to a space in the group separator.
![image](https://user-images.githubusercontent.com/26430136/198960224-41756dc4-db98-41a4-88af-efcfe787d6d4.png)

**Comment by volkanceylan** [comment]
I think the autoNumeric editor we use does not support space as a group separator, try setting group sep to something else via ScriptCulture.
